### PR TITLE
Chore/feedback

### PR DESCRIPTION
**Pull request by disturbing**
This change is [Reviewable](https://reviewable.io/reviews/kintohub/example-template/1)

### CODE REVIEW

**Comment by keblodev** [comment]
Screen Shot 2019-11-04 at 9 55 54 AM

what is _reviewable_ @Disturbing  ?

File: README.md [hunk 1]
```diff
@@ -3,62 +3,69 @@
...
-TODO: Provide Instructions via Build Args and/or Kinto Configuration for production use. Ensure each explain tells why it makes it production ready (in one sentence - dont do lengthy!)
+## Test
+
+1. npm test
 
 
 ## Usage
+The following works with our [Live Example](live/example/endpoint):
+
+GET http://localhost:8080/hello
```
Comment (keblodev) [inline README.md:50]: should be wrapped in `code` quotes

File: README.md [hunk 2]
```diff
@@ -3,62 +3,69 @@
...
+
+response:
+{
+"hello": "world"
+}
```
Comment (keblodev) [inline README.md]: same here

**Comment by edwardkcyu** [comment]
> what is _reviewable_ @Disturbing ?

I don't know too

**Comment by disturbing** [comment]
Deleting reviewable ;-p

**Review by edwardkcyu — APPROVED** [review]
LGTM, let's start using it

**Pull request merged**